Gopkg.toml

```diff
@@ -1,6 +1,6 @@
 [[constraint]]
   name = "github.com/pulumi/pulumi"
-  version = "v0.15.1"
+  revision = "e498cab2390aa24546fbc3d7950c4800b0ba3e5d"
```

Comment:
@ellismg Is there a more appropriate way to do this?  I need the latest pulumi/pulumi bits to fix a test harness issue.

Reply:
Done!